src/components/faqList/FAQList.jsx

```diff
@@ -0,0 +1,105 @@
+import {useState} from 'react';
+import './style.css'; //import css file for styling
+
+function FAQList(faqs) {
+  const [activeIndex, setActiveIndex] = useState(null); // state to store the active index of the clicked question
+
+  const handleClick = (index) => {
+    setActiveIndex(index === activeIndex ? null : index); // update activeIndex state based on the clicked index
+  };
+
+  return (
+    <div className="faq-list">
+      {faqs.map((faq, index) => (
+        <div className="faq-item" key={index}>
+          <div className="faq-question" onClick={() => handleClick(index)}>
+            <div className="faq-icon">
+              {activeIndex === index ? (
+                <span>&#x25BC;</span>
+              ) : (
+                <span>&#x25B6;</span>
+              )}
+            </div>
+            <div className="faq-text">{faq.question}</div>
+          </div>
+          {activeIndex === index && <div className="faq-answer">{faq.answer}</div>}
+        </div>
+      ))}
+    </div>
+  );
+}
+
+export function FAQTSWebsite() {
+  const faqs = [
+    {
+      question: 'Frontend Developer',
+      answer: 'Help us implement the Talente Spenden website concept using HTML and Javascript.',
+    },
+    {
+      question: 'Backend Developer',
+      answer:
+        'Create and maintain the databases that power the Talente Spenden website.',
+    },
+  ];
+  return FAQList(faqs);
+}
+
+export function FAQProjectEdu() {
+  const faqs = [
+    {
+      question: 'Education Consultant',
+      answer: 'Support us with conducting coding projects at schools. [TBD]',
+    },
+  ];
+  return FAQList(faqs);
+}
+
+export function FAQCFSWebsite() {
```

Comment:
I'd rather not have these as other components exported from this file as they are just application of your actual faqlist component. I'd copy these configs in another file (something like config.jsx) inside the respective page's folder, in this case the apply page. There I would import only FAQList from this file and pass the imported configs from the newly created config file to the component.; So, please reuse faqlist as a component and just pass different props to it in the apply page component. This makes more readable in the apply page (as the components are the same, but just with other configs).